src/components/layout/CardWithSideImage/styles.tsx

```diff
@@ -0,0 +1,35 @@
+import styled, { css } from 'styled-components'
+import tw from 'twin.macro'
+import { Col } from '../Grid'
+
+export const StyledImageCol = styled(Col)<{
+  reverseColumnsWhenStacked: boolean
+  alignImage: 'start' | 'center' | 'end'
+  justifyImage: 'start' | 'center' | 'end'
+}>`
+  ${tw`flex`}
+  ${({ reverseColumnsWhenStacked, alignImage, justifyImage }) => css`
+    ${reverseColumnsWhenStacked ? tw`order-last md:order-first` : ''}
+    ${alignImage === 'start' ? tw`items-start` : ''}
+    ${alignImage === 'center' ? tw`items-center` : ''}
+    ${alignImage === 'end' ? tw`items-end` : ''}
+    ${justifyImage === 'start' ? tw`justify-start` : ''}
```

Comment:
I would use regular css instead of tailwind here to simplify it

Reply:
Good point! 🙌🏻 
Applied the changes in a new commit.